Dockerfile.gce_vm_image_import

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/gcp-runtimes/ubuntu_16_0_4:latest
```

Comment:
What's the point of having both:
FROM gcr.io/distroless/base
FROM scratch

Also, when having FROM scratch in Docker file, running it fails with:
standard_init_linux.go:190: exec user process caused "no such file or directory"

But then it's hard to debug as no ssh.

Is this docker file acceptable?

FROM gcr.io/distroless/base

COPY linux/gce_vm_image_import /gce_vm_image_import
COPY daisy_workflows/ /daisy_workflows/

WORKDIR /
ENTRYPOINT ["/gce_vm_image_import"]

Reply:
I'm ok with this for now but you should move to a scratch container as there are still dependencies here. The error you have is from runc I believe but could just be a caught error, you should be able to copy exactly what we have with daisy, are you reading something from the filesystem other than the daisy workflows?

The point of the two FROM entries is so that we can pull ca-certs from the distroless container
COPY --from=0 /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
-from=0 specifies the first FROM, but the image is still a scratch image, no additional base layers.

Reply:
I'm not reading any other files from local file system except workflow files. The failure with scratch image as start is:

a24c3fcc3f69: Pull complete
Digest: sha256:b9a8c848c57e2a4cb35f8691318fb41cce012c33a35a5759f4cbb018a637e188
Status: Downloaded newer image for gcr.io/zoran-playground/gce_vm_image_import:latest
standard_init_linux.go:190: exec user process caused "no such file or directory"

Do we need ca-certificates file for the importer? What are they used for?

Reply:
cacerts are used for any https network connection